editor/scene_tree_dock.cpp

```diff
@@ -1599,6 +1731,7 @@ void SceneTreeDock::replace_node(Node *p_node, Node *p_by_node) {
 void SceneTreeDock::set_edited_scene(Node *p_scene) {
 
 	edited_scene = p_scene;
+	copied_nodes.clear();
```

Comment:
1) It feels to me this will memleak, you should do a proper function to get rid of the nodes when you no longer need them
2) Copying/Pasting nodes between scenes does look like a desired feature

Reply:
you may also want to call the clear clipboard funciton on exit (NOTIFICATION_EXIT_TREE)

Reply:
But only if node stuff is being copied. Why not copy serialized data instead?

Reply:
I side with @Zylann, serializing is the right way to go IMHO. BTW the reason I only copied note paths in https://github.com/godotengine/godot/pull/14810 was because keeping track of which `Node*` in an array are still valid for copying after some time (and scene changes) is a nightmare.

Reply:
I created this PR while I was learning basics.

And, yeah @Zylann  and @poke1024, I get that serialisation will be best.

I was earlier planning to write a function like `_parse_nodes`(to serialize) from the Scene State, 
And probably just keep the signals between the copied nodes too and ignore the rest.
After that I could de-serialise on paste and the rest was simple.

I was still waiting for the right time to implement it, as it's a little new for me. And as, this will likely result in me redoing the whole copy paste feature. 
And I was tight on time these days, as I just got busy with my College, just got in.

And from what I have been told, I think that without multi-scene or multi-instance copy-paste it's useless.

Reply:
@reduz there is if another unrelated part of the editor destroys them.